hacks/ops-pod

```diff
@@ -25,6 +25,7 @@ Usage:
 
 Options:
   -n|--namespace   The namespace into which the pod will be deployed. The namespace of the current kubectl context is used by default.
+  -t|--toleration  Adding toleration to the pod ex.: key1=value1:key2=value2
```

Comment:
This no longer conforms to the functionality of the script since the `--toleration` flag will copy the tolerations from the node spec.